packages/api/src/controllers/stream.ts

```diff
@@ -1375,10 +1382,17 @@ app.patch("/:id/record", authorizer({}), async (req, res) => {
     res.status(400);
     return res.json({ errors: ["can't set for session"] });
   }
+  const record = req.body.record;
   if (req.body.record === undefined) {
```

Comment:
WDYT of adding a check if `record` field is a boolean here as well? Otherwsie we can end up using different values in different places (there's a hacky conversion to boolean on line 1398)
```suggestion
  if (typeof record !== 'boolean') {
```

Reply:
Yeah, good idea.